examples/kubernetes/templates/v1/cilium-cm.yaml

```diff
@@ -155,3 +155,10 @@ data:
   # Enable legacy services (prior v1.5) to prevent from terminating existing
   # connections with services when upgrading Cilium from < v1.5 to v1.5.
   enable-legacy-services: "false"
+
+  # Enable chaining with another CNI plugin
```

Comment:
This requires documentation changes as flannel will not work if `cni-chaining-mode` is not set with `flannel`

Reply:
Why does it not work?